06-TimeManagement/module/mymodule.c

```diff
@@ -130,10 +173,11 @@ static int __init init_mod(void)
 	}
 
 	dev_info(&pdev, "HR Timer module installing\n");
-	ktime = ktime_set(0, MS_TO_NS(1000));
+	ktime = ktime_set(0, MS_TO_NS(TIMER_DELAY_MS));
 	hrtimer_init(&hr_timer, CLOCK_MONOTONIC, HRTIMER_MODE_REL);
 	hr_timer.function = &my_hrtimer_callback;
-	dev_info(&pdev, "Starting timer to fire in %dms (%lu)\n", 1000, jiffies);
+	dev_info(&pdev, "Starting timer to fire in %dms (%lu)\n",
+		1000, jiffies);
```

Comment:
It is better to use constant instead of hard-coded value 1000.

Reply:
Yes, sure, I missed it. I will be more attentive.